routing/edge_estimator.cpp

```diff
@@ -101,6 +102,14 @@ double CarEdgeEstimator::CalcHeuristic(m2::PointD const & from, m2::PointD const
 {
   return TimeBetweenSec(from, to, m_maxSpeedMPS);
 }
+
+double CarEdgeEstimator::GetUTurnPenalty() const
+{
+  // Adds 2 minutes penalty for U-turn. The value is quite arbitrary
+  // and needs to be properly selected after a number of real-world
+  // experiments.
+  return 2 * 60;  // seconds
```

Comment:
> as idea - for more accurate estimation uturn penalty can depend of counter-road traffic.

@kshalnev Could you please clarify your idea? 

> Also, it may depend on the road-type.

Let's think how it can depend on road type. I guess it's easier to make a uturn on big road because in most cases there's a special lanes for such maneuver in this case. What do you think?